### PR TITLE
Always apply single promotion rule

### PR DESCRIPTION
**Pull request by korycins**

### CODE REVIEW

File: docs/developer/discounts/promotions.mdx [hunk 1]
```diff
@@ -19,9 +19,16 @@ The promotion discount will be also visible on the checkout line prices after ad
 product to the card. The checkout subtotal is calculated as a sum of discounted prices of variants
 added to the cart.
 
+###### Saleor 3.17 (Deprecated)
+
 When multiple promotions apply to one product, the promotion with the greatest savings is used.
 The discounts from rules within the single promotion are summed up.
```
Comment (IKarbowiak) [inline docs/developer/discounts/promotions.mdx:25]: Should we leave in docs what was the previous behaviour? Maybe we should mention this change only in the upgrade guide, and here in the promotion page stay with the current behavior?

**Review by 2can — APPROVED** [review]

**Comment by IKarbowiak** [comment]
I think that we should also mention this change in the upgrade guide for 3.19

File: docs/developer/discounts/promotions.mdx (hunk 1 above)
Comment (korycins) [reply docs/developer/discounts/promotions.mdx:25]: I did this in the same way as we have for transactionAPI: https://docs.saleor.io/docs/3.x/developer/payments#saleor-34-deprecated

**Comment by korycins** [comment]
@IKarbowiak Done! 👮‍♀️

**Review by IKarbowiak — APPROVED** [review]

**Pull request merged**